src/applications/ask-va/config/schema-helpers/personalInformationHelper.js

```diff
@@ -25,6 +25,9 @@ const ssnServiceInfo = (
   <>
     <h4 className="vads-u-font-weight--bold vads-u-font-family--serif">
       Social Security or service number
+      <span className="form-required-span vads-u-font-weight--normal">
+        (*Required)
+      </span>
```

Comment:
I think it would be good to update the error state while you're here to match figma: 

![image](https://github.com/user-attachments/assets/9b3a7c7b-92f4-43b4-85d3-7a7bc41aff03)

Reply:
updated message